src/targetCalculation.mli

```diff
@@ -0,0 +1,4 @@
+open Common
+open FixedPoint
+
+val compute_current_target : (* current_q *) fixedpoint -> (* current_index *) fixedpoint -> (* current_kit_in_tok *) ratio -> fixedpoint
```

Comment:
I actually didn't realize that you could make inline comments in function signatures like this